civis/_utils.py

```diff
@@ -1,6 +1,16 @@
+from __future__ import absolute_import
+from builtins import super
+
 import re
 import uuid
 
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util import Retry
+
+import civis
+from civis import __version__ as client_version
```

Comment:
The explicit version import is really unneeded. I put that in when debugging an import error and forgot to remove it. Fixed.